src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -703,11 +709,13 @@ public static IEnumerable<TSource> SkipWhile<TSource>(this IEnumerable<TSource>
 
         private static IEnumerable<TSource> SkipWhileIterator<TSource>(IEnumerable<TSource> source, Func<TSource, bool> predicate)
         {
-            bool yielding = false;
-            foreach (TSource element in source)
+            using (IEnumerator<TSource> e = source.GetEnumerator())
```

Comment:
Wait, it saying short named locals are not acceptable from coding standard perspective? 
If so, that is fuckin ridiculous.